shimmer/learn/governance/governance-tech.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Governance voting in Shimmer Explained
+description: A technical description of the Governance functionality in Shimmer
+image: /img/learn/governance/vote-graphic04.png
+keywords:
+  - Documentation
+  - Firefly
+  - Hornet INX
+  - Voting
+  - Shimmer
+  - Governance
+---
+
+On-chain governance is important for any decentralized DLT. With Shimmer's Firefly Wallet, Shimmer token holders can participate in the governance of the network. 
+With the support of the IOTA Foundation, the Shimmer community has developed a highly flexible and secure system for casting votes directly on the Shimmer Tangle 
+through the Firefly Wallet and counting the votes in the network’s Hornet nodes.
+
+## Voting Phase
+
+The Voting phase of a governance vote is further divided into four stages:
+
+- **Proposal announcement:** Upon publishing the voting ballot to the [GitHub repository](https://github.com/iota-community/Shimmer-governance-participation-events), Hornet node operators can activate the voting ballot in their nodes’ Participation Plugin to be prepared to count any votes that are cast.
+- **Voting Open:** The Voting Open stage lasts seven days and starts when a predefined milestone in the Shimmer network is reached. After the network has passed this milestone, users can cast, change, revoke, or redo the vote using the governance functionality of the Shimmer Firefly wallet. It's important to note that changes of opinion during this timeframe will not affect the final vote count. Ideally, voters should cast their votes before the next stage begins to ensure the highest possible vote count.
```

Comment:
I think this should be Open Voting

Reply:
Yeah, but its officially called like this in the Wallet etc, so I'll keep it consistent with what people see